src/storage/blockchain/state.ts

```diff
@@ -34,6 +36,8 @@ const defaultEthConnectionConfig: EthConnectionConfig = {
   defaultGasLimit: 600000,
   minGasPrice: '0',
   maxGasPrice: '100000000000',
+  maxFeePerGas: '', //'2000000000',
```

Comment:
maybe we don't need to init it at all? not with empty string (same for priority fee)